src/classes/drawings/drawing.vala

```diff
@@ -91,6 +91,8 @@ namespace pdfpc.Drawings {
         private int current_slide {get; set;}
         private Drawings.Storage.Base storage {get; protected set;}
 
+        private Drawings.DrawingCommands.DrawingCommandList drawing_command_list;
```

Comment:
I guess there is no need to keep the the upper-level `Drawings` in the class hierarchy.